docs/shared/react-tutorial/react-monorepo.md

```diff
@@ -0,0 +1,827 @@
+
+/* eslint-disable-next-line */
+export interface ProductListProps {}
+
+export function ProductList(props: ProductListProps) {
+  return (
+    <div className={styles['container']}>
+      <h1>Welcome to ProductList!</h1>
+    </div>
+  );
+}
+
+export default ProductList;
+```
+
+Make sure the `ProductList` is exported via the `index.ts` file of our `products` library. This is our public API with the rest of the workspace. Only export what's really necessary to be usable outside the library itself.
+
+```ts {% fileName="libs/products/src/index.ts" %}
+export * from './lib/product-list/product-list';
+```
+
+We're ready to import it into our main application now. First (if you haven't already), let's set up React Router.
+
+```shell
+npm install react-router-dom
+```
+
+Configure it in the `main.tsx`.
+
+```tsx {% fileName="src/main.tsx" %}
+import { StrictMode } from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import ReactDOM from 'react-dom/client';
+
+import App from './app/app';
+
+const root = ReactDOM.createRoot(
+  document.getElementById('root') as HTMLElement
+);
+
+root.render(
+  <StrictMode>
+    <BrowserRouter>
+      <App />
+    </BrowserRouter>
+  </StrictMode>
+);
+```
+
+Then we can import the `ProductList` component into our `app.tsx` and render it via the routing mechanism whenever a user hits the `/products` route.
+
+```tsx {% fileName="src/app/app.tsx" %}
+import { Route, Routes } from 'react-router-dom';
+
+// importing the component from the library
+import { ProductList } from '@react-monorepo/products';
+
+function Home() {
+  return <h1>Home</h1>;
+}
+
+export function App() {
+  return (
+    <Routes>
+      <Route path="/" element={<Home />}></Route>
+      <Route path="/products" element={<ProductList />}></Route>
+    </Routes>
+  );
+}
+
+export default App;
+```
+
+Serving your app (`nx serve myreactapp`) and then navigating to `/products` should give you the following result:
+
+![products route](/shared/images/tutorial-react-standalone/react-tutorial-products-route.png)
+
+Let's apply the same for our `orders` library.
+
+- generate a new component `OrderList` in `libs/orders` and export it in the corresponding `index.ts` file
+- import it into the `app.tsx` and render it via the routing mechanism whenever a user hits the `/orders` route
+
+In the end, your `app.tsx` should look similar to this:
+
+```tsx {% fileName="src/app/app.tsx" %}
+import { Route, Routes } from 'react-router-dom';
+import { ProductList } from '@react-monorepo/products';
+import { OrderList } from '@react-monorepo/orders';
+
+function Home() {
+  return <h1>Home</h1>;
+}
+
+export function App() {
+  return (
+    <Routes>
+      <Route path="/" element={<Home />}></Route>
+      <Route path="/products" element={<ProductList />}></Route>
+      <Route path="/orders" element={<OrderList />}></Route>
+    </Routes>
+  );
+}
+
+export default App;
+```
+
+## Building the App for Deployment
```

Comment:
I think we should explain more here, like how we handle deployments for both apps. That is..

- deploy both individually => running the build for both individually
- just run `npx nx run-many -t build -p myreactapp second-app`
- use `affected:build`

for the "affected" to work here, we need to have shown it before. I forgot adding a section for that on the Linear issue, but I'd add that after the "Testing and Linting...." where we show running multiple commands and caching. I'd have a section right after that shows how to only run it for a subset of the monorepo, show the affected part n the graph etc...